canvas_sdk/commands/commands/task.py

```diff
@@ -52,7 +44,7 @@ def values(self) -> dict:
         """The Task command's field values."""
         return {
             "title": self.title,
-            "assign_to": self.assign_to.as_dict() if self.assign_to else None,
+            "assign_to": self.assign_to if self.assign_to else None,
```

Comment:
```suggestion
            "assign_to": self.assign_to,
```
Unless `assign_to` can be something else other that `None`, this should work right?